### ISSUE TITLE
question/ntr: Perimenopause and postmenopause

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7603]
The INCLUDE DCC (cc @lopierra) needs to annotate clinical data with Perimenopause and postmenopause. Would these be appropriate terms for HPO? I wasn't sure, as these are normal biological processes.

**Comment by pnrobinson** [comment]
These are similar to the current term puerperal onset, so they are in scope.

**Comment by nicolevasilevsky** [comment]
Thanks @pnrobinson, I'll create a PR to add these terms

**Comment by pnrobinson** [comment]
Hi Nicole, It would be great to have an HPO wide meeting to discuss better structuring of the onset and inheritance terms -- we have gotten a lot of good feedback in this area. Perhaps @mellybelly @cmungall @matentzn Moni and the curation team could meet for an hour at some upcoming slot?

**Comment by nicolevasilevsky** [comment]
Ok! Would you like this meeting to happen before we add these new terms?

**Comment by pnrobinson** [comment]
Yeah, I think we now have two kinds of onset -- temporal and situational and so maybe we need more structure!

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000314
- label: Perimenopausal onset
- parents: [Onset (HP:0003674)](https://hpo.jax.org/app/browse/term/HP:0003674)
- pmid: 
- def: Onset of signs or symptoms of disease during the period of a woman's life shortly before the occurrence of the menopause.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7603
- synonyms:

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000315
- label: Postmenopausal onset
- parents: [Neonatal onset (HP:0003623)](https://hpo.jax.org/app/browse/term/HP:0003623)
- pmid: 
- def: Onset of signs or symptoms of disease during the period of a woman's life after the occurrence of the menopause.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7603
- synonyms:
